andela/vale-ah-backend
# #164139773 Create User Profile

#### What does this PR do?
Creates User Profile
Creates User Login
#### Description of Task to be completed?
- write unit test
- Create user profile route
- Create user login route
- validate inputs with `joi`
#### How should this be manually tested?
- ensure that the build is passing on travis 
- Test endpoint api/users/login
- Test endpoint api/user
#### What are the relevant pivotal tracker stories?
[#164139773](https://www.pivotaltracker.com/story/show/164139783)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
